wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java

```diff
@@ -39,6 +39,9 @@
 
   public static final String CHE_WORKSPACE_AUTO_START = "che.workspace.auto_start";
 
+  /** This attribute describes latest startup error message */
+  public static final String ERROR_MESSAGE_ATTRIBUTE_NAME = "errorMessage";
```

Comment:
It would be better to rename this attribute to make it clear that it is startup error message.

Reply:
It probably can be used not only for startup errors

Reply:
Then please fix java doc for it